app/controllers/api/markdown_controller.rb

```diff
@@ -0,0 +1,9 @@
+module API
+  class MarkdownController < BaseController
+    def parse
+      html = ParseMarkdown.(params[:markdown].to_s)
+
+      render json: { html: html }
+    end
+  end
+end
```

Comment:
Pls add a controller test that checks calling it with markdown returns html and is a 200.